metrics/src/test/java/com/expedia/adaptivealerting/metrics/functions/source/graphite/GraphiteQueryResultTest.java

```diff
@@ -0,0 +1,81 @@
+package com.expedia.adaptivealerting.metrics.functions.source.graphite;
+
+import lombok.val;
+import org.junit.Test;
+import java.util.Iterator;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class GraphiteQueryResultTest {
+
+    @Test
+    public void testgetGraphiteQueryResultFromJson(){
+       val sampleJsonGraphite = "[{" + "\"datapoints\": [[12.0, 1568255056]], " +
+               "\"target\": \"sumSeries(a.b.c)\", " +
+               "\"tags\":" + "{" + "\"aggregatedBy\": \"sum\", \"name\": \"sumSeries(a.b.c)\"" + "}}]";
```

Comment:
Consider creating an actual class to represent this Graphite data type, and then using Jackson to serialize it to text. This is easier to manage than hand-crafting strings with all the escaped quotes.

Reply:
Added changes to create test String from Object and not hand crafted string.